terraform/bootstrap/remote-state/production/main.tf

```diff
@@ -0,0 +1,8 @@
+module "production_tf_state" {
+    source = "../../../modules/remote-state"
+    environment = "production"
```

Comment:
We already have the environment provided as a variable when apply is run. How about just use that var here so we don't have to duplicate the files?

Reply:
as in not moving it to a module?
I would still need 1 set of the file per env, right? 

or you mean just using `environment = var.environment`

Reply:
Yeah, just using the variable.

Reply:
Actually I think even having it as a module is overkill. How about just leave it as it was before, but accept the environment variable?

Reply:
sure, either way is easy to change down the road